clients/ts/webdriver-tap-runner/package-lock.json

```diff
@@ -1099,6 +1099,14 @@
         }
       }
     },
+    "string_decoder": {
```

Comment:
Is this relevant or are you going to revert this?

Reply:
I believe it happens when the version of NPM is different maybe? I'm going to update to the latest released NPM and run `npm i` to see if that fixes it (my home desktop hasn't been updated in a while). If that doesn't fix it, I'll presume that this is the "new" ordering and we should check it in anyway.

Reply:
Yep. Updated NPM and ran `./build /t:Restore` and everything went back to normal.